test/support/utils.rb

```diff
@@ -64,6 +64,7 @@ def create_pseudo_terminal(boot_options: "-r debug/run")
                   cmd = @queue.pop
                 end
                 write.puts(cmd)
+                @last_backlog = []
```

Comment:
perhaps Array#clear conveys the intention better?

Reply:
I think so, too. Thanks.

Reply:
https://github.com/ruby/debug/pull/109/files#diff-ee14bf65f14927f7778a246f90c5cec0c581be3e2658c2a8a3726c59db5be3f9R67